addons/account/models/account_move.py

```diff
@@ -2763,6 +2776,10 @@ def action_duplicate(self):
         action['res_id'] = self.copy().id
         return action
 
+    def action_activate_currency(self):
+        self.ensure_one()
+        self.currency_id.active = True
```

Comment:
Make it works in multi: `self.currency_id.filtered(lambda currency: not currency.active).write({'active': True})`